data-table/__tests__/table.js

```diff
@@ -1,9 +1,65 @@
+// test 4-5 function
 const table = require('../table.js');
 
 beforeEach(() => {
     document.body.innerHTML = '<div id="div-id"></div>'
 });
 
+describe('test createDataTable', () => {
+      
+  test('test0', () => {
```

Comment:
For this and all other descriptions (`test0`, `test1`, etc) have more complete descriptions of what the test really does